cms/tests/test_menu.py

```diff
@@ -297,6 +297,29 @@ def test_basic_cms_menu(self):
             [page.get_absolute_url() for page in pages],
         )
 
+    def test_show_new_draft_page_in_menu(self):
+        with self.login_user_context(self.get_superuser()):
+            response = self.client.post(URL_CMS_PAGE_ADD, self.get_new_page_data())
+            self.assertRedirects(response, URL_CMS_PAGE)
+
+        request = self.get_request('/')
+        renderer = menu_pool.get_renderer(request)
+        renderer.draft_mode_active = True
+        renderer.get_nodes()
+        self.assertEqual(CacheKey.objects.first().key, 'cms_3.6.0rc1_menu_nodes_en_1:draft')
```

Comment:
not going to work once we release new versions because of the hardcoded version number.
have a look at other menu tests, I know we have checked in the past whether we're being cached.